ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml

```diff
@@ -382,6 +382,13 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private
+- as: ibmcloud-ipi-dedicatedhost-f28
```

Comment:
I think we do not need this in stable ci.

Reply:
I will remove it from [openshift-openshift-tests-private-release-4.14__amd64-stable.yaml](https://github.com/openshift/release/pull/43123/files/1142465ed9565ab756a4fcb189a58cd6741749d2#diff-81032cdc5b72c2c63c4e966241c6f754881feffe1564c691411fe138681bf4c2) after review, just for skip e2e test